plugins/com.cubrid.common.ui/src/com/cubrid/common/ui/spi/Messages.properties

```
@@ -191,7 +191,7 @@ tipRecommendDriver=This database version is {0}.\nPlease use latest the JDBC dri
 tipNoSupportJdbcVersion=This database version is {0}. However, the version of the selected JDBC driver is different and therefore it may not completely support this database.\r\nDo you want to continue?
 errCommonTip=Error code: {0}\nError Message: {1}
 
-errLockNoUseTemporary=CUBRID Admin is not yet supported retrieving of Lock informations on CUBRID 9.1.\nPlease you run "cubrid lockdb {dbname}" on the command window.
+errLockNoUseTemporary=Unable to display lock informations.\nPlease you run "cubrid lockdb {dbname}" on the command window.
```

```suggestion
Unable to display lock information.\nPlease run "cubrid lockdb {dbname}" from the command line.
```
nit: 단순히 조금 더 자연스러운 표현이 어떨까 제안합니다!..